gradio/components/audio.py

```diff
@@ -49,7 +49,7 @@ class Audio(
 ):
     """
     Creates an audio component that can be used to upload/record audio (as an input) or display audio (as an output).
-    Preprocessing: passes the uploaded audio as a {Tuple(int, numpy.array)} corresponding to (sample rate in Hz, audio data as a 16-bit int array whose values range from -32768 to 32767), or as a {str} filepath, depending on `type`.
+    Preprocessing: depending on `type`, passes the uploaded audio as {str} filepath or a {Tuple(int, numpy.array)} corresponding to (sample rate in Hz, audio data). If the latter, the audio data is a 16-bit int array whose values range from -32768 to 32767 and shape of the audio data array is (samples,) for mono audio or (samples, channels) for multi-channel audio.
```

Comment:
```suggestion
    Preprocessing: depending on `type`, passes the uploaded audio as {str} filepath or a {Tuple(int, numpy.array)} corresponding to (sample rate in Hz, audio data). If the latter, the audio data is a 16-bit int array whose values range from -32768 to 32767 and shape of the audio data array is {(samples,)} for mono audio or {(samples, channels)} for multi-channel audio.
```

Reply:
Why the curly brackets?

Reply:
Adds formatting that makes it easier to parse the text. Feel free to ignore

<img width="841" alt="Screen Shot 2024-01-24 at 11 00 35 AM" src="https://github.com/gradio-app/gradio/assets/9021060/e9edcd07-ad06-42b7-9d35-efe97acf78dc">

<img width="811" alt="Screen Shot 2024-01-24 at 11 03 35 AM" src="https://github.com/gradio-app/gradio/assets/9021060/c6d39f6e-0667-4b0b-9fdd-4bdbab6969c1">

Reply:
I thought we only used that to designate Python types